library/spdm_transport_mctp_lib/libspdm_mctp_common.c

```diff
@@ -250,7 +250,7 @@ libspdm_return_t libspdm_transport_mctp_decode_message(
             libspdm_get_secured_message_context_via_session_id(
                 spdm_context, *secured_message_session_id);
         if (secured_message_context == NULL) {
-            spdm_error.error_code = SPDM_ERROR_CODE_INVALID_SESSION;
+            spdm_error.error_code = SPDM_ERROR_CODE_DECRYPT_ERROR;
```

Comment:
You changed the behavior.

`SPDM_ERROR_CODE_INVALID_SESSION` means to ignore, not response.

`SPDM_ERROR_CODE_DECRYPT_ERROR` may return error message based upon `LIBSPDM_DATA_HANDLE_ERROR_RETURN_POLICY_DROP_ON_DECRYPT_ERROR`.

I suggest we may keep `SPDM_ERROR_CODE_INVALID_SESSION` to internal at first, because we still need a flag to tell other module, that it gets an invalid session.

Reply:
It seems like `SPDM_ERROR_CODE_INVALID_SESSION` needs to remain since it's a breaking change. I can move it to `library/spdm_secured_message_lib.h` though so that it's out of `spdm.h`.